pkg/services/oracle/neofs/neofs.go

```diff
@@ -45,41 +48,47 @@ var (
 // URI scheme is "neofs:<Container-ID>/<Object-ID/<Command>/<Params>".
 // If Command is not provided, full object is requested.
 func Get(ctx context.Context, priv *keys.PrivateKey, u *url.URL, addr string) (io.ReadCloser, error) {
-	objectAddr, ps, err := parseNeoFSURL(u)
+	c, err := GetSDKClient(ctx, addr)
 	if err != nil {
-		return nil, err
+		return clientCloseWrapper{c: c}, fmt.Errorf("failed to create client: %w", err)
 	}
+	return GetWithClient(ctx, c, priv, u, true)
+}
 
-	c, err := client.New(client.PrmInit{})
+// GetWithClient returns a neofs object from the provided url using the provided client.
+// URI scheme is "neofs:<Container-ID>/<Object-ID/<Command>/<Params>".
+// If Command is not provided, full object is requested.
+func GetWithClient(ctx context.Context, c *client.Client, priv *keys.PrivateKey, u *url.URL, wrapClientCloser bool) (io.ReadCloser, error) {
```

Comment:
Move this client modifications to a separate commit. And wrapClientCloser should be documented.